Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -63,7 +64,12 @@
     - id: ClothingHeadCage # backmen
     - id: HandheldReinforcementCentCom # backmen
 #    - id: ClothingEyesGlassesCommand # Backmen - Because dublicate (56)???
-
+    - id: ClothingEyesGlassesCommand
+    - id: HeadSkeleton # A skull to accompany your skeleton crew
+      conditions:
+      - !type:PlayerCountCondition
+        max: 15
+     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)